inst/templates/ad_adlb.R

```diff
@@ -0,0 +1,328 @@
+# Name: ADLB
+#
+# Label: Lab Analysis Dataset
+#
+# Input: adsl, lb
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(dplyr)
+library(lubridate)
+library(stringr)
+
+# ---- Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+data("lb")
+data("adsl")
+
+lb <- convert_blanks_to_na(lb)
```

Comment:
Can we put in a tiny blurb about why we do this?

Reply:
I was thinking more of why do even do this step?  Why not just leave them as blanks?

Reply:
I think if you just pass in blank strings R will treat it as a string, instead of a proper missing value. So we need `convert_blanks_to_na()` to convert them to missing values for the downstream functions to work properly. The "Get Started" page mentioned something similar in the "Handling of Missing Values" section.

Reply:
Maybe just put the url in for Why this step?  See Handling of Missing Values - 
 https://pharmaverse.github.io/admiral/articles/admiral.html#handling-of-missing-values

Reply:
We also have this from Programming Strategy.

[](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#file-structuring)Missing values
Missing values (NAs) need to be explicitly shown.

Regarding character vectors converted from SAS files: SAS treats missing character values as blank. Those are imported into R as empty strings ("") although in nature they are missing values (NA). All empty strings that originate like this need to be converted to proper R missing values NA.

If we add in a small blurb, then we shold do the same to the other BDS scripts, ADVS, ADEG.

Reply:
I will make this a separate issue for others to think about and apply this update to all the templates.  #1093